File: Day220/reverseAString.java
package Day220;
public class reverseAString{
    public static String reverseWord(String str)
    {
        // Reverse the string str
        int low = 0;
        int high = str.length()-1;
        char arr[] = str.toCharArray();
        while (low <= high) {
            char temp = arr[low];
            arr[low]= arr[high];
            arr[high] = temp;
            low++;
            high--;
        }
        return new String(arr);
    }
}

File: Day220/reverseAString.go
package main 

import (
	"fmt"
)

func reverse(s string) string {
	low := 0
	high := len(s) - 1
	arr:= make([]byte, len(s))
	for i:=0; i<len(s); i++ {
		arr[i] = byte(s[i])
	}
	for low <= high {
		arr[low] = s[high]
		arr[high] = s[low]
		low++
		high--
	}
	s = ""
	for i:=0; i<len(arr); i++ {
		s += string(arr[i])
	}
	return s
}

func main() {
	s := "hello"
	fmt.Println(reverse(s))
}